javascript/angular/security/detect-angular-trust-as-js-method.yaml

```diff
@@ -2,9 +2,13 @@ rules:
   - id: detect-angular-trust-as-js-method
     patterns:
       - pattern-either:
-          - pattern: |
-              $SOURCE = $scope.$INPUT;
-              $sce.trustAsJs($SOURCE);
+          - patterns:
+            - pattern-inside: |
+                $SOURCE = $scope.$INPUT;
+                ...
+                $sce.trustAsJs($SOURCE);
```

Comment:
These actually should probably be taint rules at some point in the future. 
Having only 
```
 - pattern: | 
     $sce.trustAsJs($INPUT);
```
does not verify if this input is originating as user input. i also changed to rule to use a combination of `pattern-inside` and `pattern` so that I could rewrite the right part with the fix. But the fixes of these angular rules are going to be removed anyway, because @LewisArdern agreed that the current fix is probably not a good idea anyway. So I am going to revert these rules to their previous state.